spaces/spaceswap/skin.scss

```
@@ -0,0 +1,10 @@
+.name {
```

The css selector should similar to the space key.
```suggestion
.spaceswap {
```

---

dear e-asia - I fixed this

---

please, merge